### ISSUE TITLE
Unity recorder failed recording depth sequences

### ISSUE THREAD

**Issue by TianCuteQY** [CapsuleEndoscope/VirtualCapsuleEndoscopy#7]
Hi! I was trying to record depth image sequences as described in README.md, but they came out as RGB rendered images. Here are how I set properties in unity AOV recorder and images coming out:
![1](https://user-images.githubusercontent.com/95089546/155327672-7105118e-16fa-41a7-b8cc-73c13b043106.PNG)
![2](https://user-images.githubusercontent.com/95089546/155327688-b5dfcdbb-7c25-4ce5-94f1-4dd538abca0b.PNG)
What did I do wrong?
Thank you for your time!

**Comment by inckagan** [comment]
Hi, 
It seems that the Recorder is updated. I have checked and found the following link:
https://forum.unity.com/threads/unity-aov-recorder-update.717089/
Based on that, problem could be related to scene,
Can you check if you are using the AOVRecorder with an HDRP scene?

**Comment by TianCuteQY** [comment]
Thanks again for your reply! 
I use Unity2019.3, AOVRecorder 0.1.1 preview2 and Recorder 2.4.0 preview1. I am very new to Unity but I installed HDRP 7.1.8 so I assume I'm recording a HDRP scene. After your reply, I tried ealier versions of AOVRecorder and they also didn't work. Then I changed camera tag to main camera, depth output of AOVRecorder turns to all black. 
I'm quite stuck here and could not find similar issues online, so I assume it's caused by some of my settings. But thanks to your project, I mannaged to record motion and reproduce the depth images using vtk.
